nsv13/code/modules/munitions/ship_weapons/energy_weapons/bsa.dm

```diff
@@ -227,3 +227,10 @@
 		var/mob/living/goodbye = target
 		goodbye.dust(TRUE, FALSE)
 	explosion(get_turf(target), 6, 8, 9, 12, ignorecap = TRUE, flame_range = 6) //I have to keep myself from letting it just truncate ships because thats a bit annoying to fix for the receiving side, even if accurate to appearance.
+
+
+/obj/machinery/ship_weapon/energy/beam/bsa/after_fire()  //NT gets to cheat and have no degadation or heating on their weapon because muni has enough to do and the BSA doesn't need buffs/nerfs
+	. = ..()
+	heat = 0
+	alignment = 100
+	freq = 100
```

Comment:
I'd have to add a check for that in every part of the system though right? Would that be better

Reply:
It's the classic "address now vs. address later" issue.
For now it's probably fine with a single energy weapon that works this way, but if more appear, it may get a bit less convenient.
(Besides the fact you can do all the related maintenance steps on the BSA with this most likely despite them having no effect, and even get the examine talking about these things.)